components/prize_pool/wikis/commons/starcraft_starcraft2/prize_pool_starcraft.lua

```diff
@@ -0,0 +1,357 @@
+	}
+	local extradata = {
+		featured = Variables.varDefault('featured') or 'false',
+		lastStatuses = lastStatuses,
+		placeRange = {placement.placeStart, placement.placeEnd},
+		playernumber = StarcraftOpponent.partySize(opponent.opponentData),
+		points = placement:getPrizeRewardForOpponent(opponent, PRIZE_TYPE_POINTS .. 1),
+		points2 = placement:getPrizeRewardForOpponent(opponent, PRIZE_TYPE_POINTS .. 2),
+		seriesnumber = seriesNumber and string.format('%05d', seriesNumber) or '',
+		wofrom = lastStatuses[1] == WALKOVER_VS_STATUS or nil,
+		woto = lastStatuses[2] == WALKOVER_VS_STATUS or nil,
+	}
+
+	-- make these available for the stash further down
+	lpdbData.liquipediatier = _tier
+	lpdbData.liquipediatiertype = Variables.varDefault('tournament_liquipediatiertype')
+	lpdbData.type = Variables.varDefault('tournament_type')
+
+	lpdbData.weight = Weight.calc(
+		lpdbData.individualprizemoney or 0,
+		lpdbData.liquipediatier,
+		lpdbData.placement,
+		lpdbData.liquipediatiertype,
+		lpdbData.type
+	)
+
+	if type(lpdbData.players) == 'table' then
+		lpdbData.players = StarcraftOpponent.toLpdbStruct(opponent.opponentData).opponentplayers
+	end
+
+	if lpdbData.lastvs then
+		local lastVs = opponent.additionalData.LASTVS
+		extradata.vsOpponent = Table.deepCopy(lastVs)
+		if lastVs.type == StarcraftOpponent.team then
+			lpdbData.lastvs = Json.stringify{
+				type = lastVs.opponenttype,
+				name = lastVs.name
+			}
+		else
+			lastVs = StarcraftOpponent.toLpdbStruct(lastVs) or {}
+			lpdbData.lastvs = Json.stringify(Table.merge(
+					lastVs.opponentplayers or {},
+					{type = lastVs.opponenttype}
+				))
+		end
+	end
+
+	lpdbData.extradata = Table.mergeInto(lpdbData.extradata, extradata)
+
+	lpdbData.mode = CustomPrizePool._getMode(opponent.opponenttype, opponent.opponentData)
+	lpdbData.tournament = _tournament_name
+	lpdbData.series = _series
+
+	local prizePoolIndex = tonumber(Variables.varDefault('prizepool_index')) or 0
+	if prizePoolIndex == 1 and _tournament_extradata_cache[lpdbData.placement or ''] then
+		table.insert(_tournament_extradata_cache[lpdbData.placement], Table.deepCopy(lpdbData))
+	end
+
+	lpdbData.objectName = CustomPrizePool._overwriteObjectName(lpdbData, prizePoolIndex)
+	lpdbData.id = lpdbData.objectName -- wanted for the stash
+
+	table.insert(_lpdb_stash, Table.deepCopy(lpdbData))
+
+	return lpdbData
+end
+
+function CustomPrizePool._overwriteObjectName(lpdbData, prizePoolIndex)
+	if lpdbData.opponenttype == StarcraftOpponent.team then
+		return lpdbData.objectName .. '_' .. prizePoolIndex
+	end
+
+	return lpdbData.objectName
+end
+
+function CustomPrizePool._getMode(opponentType, opponent)
+	if (opponent or {}).isArchon then
+		return 'archon'
+	end
+
+	return OPPONENT_TYPE_TO_MODE[opponentType or '']
+end
+
+function CustomPrizePool._getStatusFromScore(score)
+	return Logic.isNumeric(score) and SCORE_STATUS or score
+end
+
+function CustomPrizePool._storeIntoTournamentLpdb()
```

Comment:
```suggestion
function CustomPrizePool._createLpdbTournament()
```

Reply:
it does not necessarily create it, it rather appends to (overwrites) the extradata
will change it to `._appendLpdbTournament()` let me know if that is not okay^^